features/input-email-tel-url.yml

```
@@ -0,0 +1,14 @@
+name: <input> with types email, tel, and url
+description: The `<input>` element allows the user to input information to a form intended for email addresses, telephone numbers or URLs. Particularly useful in combination with form validation.
```

We don't do cross references in descriptions like caniuse (rarely) does.

```suggestion
description: The `<input type="email">`, `<input type="tel">`, and `<input type="url">` HTML elements represent email address, telephone number, and URL fields in a form.
```